yaml.go

```diff
@@ -167,7 +168,7 @@ func applyYaml(org string, fileName string, format string) {
 func makeCommaSeparatedString(arr []string) string {
 	var result string
 	for _, v := range arr {
-		result += v + ","
+		result += strings.TrimRight(v, " ") + ","
```

Comment:
Maybe use TrimSpace to trim spaces from leading and trailing both?

Reply:
Done. Thanks!